src/apis/weather.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const { REACT_APP_APP_ID } = process.env;
+const instance = axios.create({
+  baseURL: "https://api.openweathermap.org/data/2.5",
```

Comment:
base url should be inside .env file.

REACT_APP_BASE_URL=https://api.openweathermap.org/data/2.5
and call it here

`baseUrl: process.env.REACT_BASE_URL`

Reply:
Done